Source/Core/DiscIO/DiscScrubber.cpp

```diff
@@ -59,7 +56,7 @@ void DiscScrubber::MarkAsUsed(u64 offset, u64 size)
   u64 current_offset = Common::AlignDown(offset, CLUSTER_SIZE);
   const u64 end_offset = offset + size;
 
-  DEBUG_LOG(DISCIO, "Marking 0x%016" PRIx64 " - 0x%016" PRIx64 " as used", offset, end_offset);
+  DEBUG_LOG_FMT(DISCIO, "Marking 0x{:016x} - 0x{:016x} as used", offset, end_offset);
```

Comment:
Any thoughts about whether `{:#018x}` is better than `0x{:016x}`?

Reply:
I have no opinion either way. I'm fine with whichever variant is comfortable for most.

Reply:
It's basically the same for me... Though I did use `#` in my PR, for what it's worth.

Reply:
Alright, I'll go with `#` then.

Reply:
You should change the `0x{:x}` ones too in that case.

Reply:
Done